.changeset/giant-eggs-occur.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
```

Comment:
Let's also add @wso2is/core, @wso2is/i18n and @wso2is/admin.claims.v1 as well

Reply:
Also let's bump console to `minor` as this is a new feature introduced to console app

Reply:
Fixed by - https://github.com/wso2/identity-apps/pull/7968/commits/43bde29076aefa01ee70595cb7d63588034d750e